homeassistant/components/smartthings/sensor.py

```diff
@@ -53,371 +56,391 @@ class SmartThingsSensorEntityDescription(SensorEntityDescription):
+    # no fixtures
+    Capability.EQUIVALENT_CARBON_DIOXIDE_MEASUREMENT: {
+        Attribute.EQUIVALENT_CARBON_DIOXIDE_MEASUREMENT: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.equivalent_carbon_dioxide_measurement,
+                key=Attribute.EQUIVALENT_CARBON_DIOXIDE_MEASUREMENT,
                 name="Equivalent Carbon Dioxide Measurement",
                 native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
                 device_class=SensorDeviceClass.CO2,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ]
     },
-    Capability.formaldehyde_measurement: {
-        Attribute.formaldehyde_level: [
+    # no fixtures
+    Capability.FORMALDEHYDE_MEASUREMENT: {
+        Attribute.FORMALDEHYDE_LEVEL: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.formaldehyde_level,
+                key=Attribute.FORMALDEHYDE_LEVEL,
                 name="Formaldehyde Measurement",
                 native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ]
     },
-    Capability.gas_meter: {
-        Attribute.gas_meter: [
+    # no fixtures
+    Capability.GAS_METER: {
+        Attribute.GAS_METER: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.gas_meter,
+                key=Attribute.GAS_METER,
                 name="Gas Meter",
                 native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
                 device_class=SensorDeviceClass.ENERGY,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ],
-        Attribute.gas_meter_calorific: [
+        Attribute.GAS_METER_CALORIFIC: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.gas_meter_calorific,
+                key=Attribute.GAS_METER_CALORIFIC,
                 name="Gas Meter Calorific",
             )
         ],
-        Attribute.gas_meter_time: [
+        Attribute.GAS_METER_TIME: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.gas_meter_time,
+                key=Attribute.GAS_METER_TIME,
                 name="Gas Meter Time",
                 device_class=SensorDeviceClass.TIMESTAMP,
                 value_fn=dt_util.parse_datetime,
             )
         ],
-        Attribute.gas_meter_volume: [
+        Attribute.GAS_METER_VOLUME: [
             SmartThingsSensorEntityDescription(
-                key=Attribute.gas_meter_volume,
+                key=Attribute.GAS_METER_VOLUME,
                 name="Gas Meter Volume",
                 native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
                 device_class=SensorDeviceClass.GAS,
                 state_class=SensorStateClass.MEASUREMENT,
             )
         ],
     },
-    Capability.illuminance_measurement: {
-        Attribute.illuminance: [
+    # no fixtures
```

Comment:
```suggestion
```

Probbaly leftover from tests. Please remove all

Reply:
I left these intentionally because I want to make sure everything is tested, but I don't have the data for this one, and I have the suspicion some are documented but never used